src/test/java/net/fabricmc/fabric/events/PlayerTickEventMod.java

```diff
@@ -0,0 +1,27 @@
+package net.fabricmc.fabric.events;
+
+import net.fabricmc.api.ModInitializer;
+import net.fabricmc.fabric.api.event.player.PlayerTickCallback;
+import net.minecraft.entity.EquipmentSlot;
+import net.minecraft.entity.effect.StatusEffectInstance;
+import net.minecraft.entity.effect.StatusEffects;
+import net.minecraft.item.ItemStack;
+import net.minecraft.item.Items;
+import net.minecraft.text.TextComponent;
+
+
+public class PlayerTickEventMod implements ModInitializer {
+	@Override
+	public void onInitialize() {
+		PlayerTickCallback.EVENT.register(player -> {
+			ItemStack stack = player.getEquippedStack(EquipmentSlot.HEAD);
+			if (stack.getItem() == Items.PLAYER_HEAD) {
+				//if someone is wearing asie's head, make them glow
```

Comment:
Tests should use java docs rather than line comments imo

Reply:
That's out of scope of this PR, multiple other test classes already use comments instead of javadocs